shared/js/content-scripts/block-cookie.js

```diff
@@ -0,0 +1,92 @@
+// Set up 3rd party cookie blocker
+(function cookieBlocker () {
+    function getFrameUrl () {
+        let url = document.location.href
+        let parentFrame = (document != window.top) && window.parent
```

Comment:
```suggestion
        let parentFrame = (document !== window.top) && window.parent
```

Is this file covered by our linting?